meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -151,6 +151,10 @@ OSTREE_KERNEL_ARGS:mx6ull-generic-bsp ?= "console=tty1 console=ttymxc0,115200 ${
 OSTREE_KERNEL_ARGS:mx7d-generic-bsp ?= "console=tty1 console=ttymxc0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
 OSTREE_KERNEL_ARGS:mx7ulp-generic-bsp ?= "console=tty1 console=ttyLP0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
 
+PREFERRED_VERSION_meson ?= "0.61.3"
+PREFERRED_VERSION_meson:imx8-nxp-bsp ?= "1.3.1.imx"
+PREFERRED_VERSION_meson:imx9-nxp-bsp ?= "1.3.1.imx"
+
```

Comment:
because we are setting here PREFERRED_VERSION, we may want to mark this somehow as a backport.
I suggest we use the [REVERTME-scargthgap] tag as we use on factories https://docs.foundries.io/latest/reference-manual/linux/linux-update.html#common-pitfalls

Reply:
we have more backports that are marked but I agree to mark it someway as it was easy to remember

Reply:
yes, I'm still trying to find good ways to track the backports

I will recreate this patch, so we can see what it looks like with the tag

Reply:
Instead of having the [REVERTME-scargthgap] tag on the commit title, I would rather open one a new PR as draft revert this one

Reply:
I think we can keep without the tag since this branch in particular is kirkstone specific, having it under recipes-backports is already enough from my perspective.

I believe we will have to keep 2 branches in order to keep kirkstone and scarthgap in place, so we would just skip this from merging in scarthgap.